deploy/jenkins-ci/Makefile

```diff
@@ -20,10 +20,22 @@ ifeq ($(RELEASE_TYPE),minor)
   # For minor, remove the -SNAPSHOT suffix from the Makefile
   BACKEND_VERSION_TO_RELEASE ?= $(shell semver bump release $(BACKEND_VERSION))
 endif
+
+ifeq ($(findstring snapshot,$(RELEASE_TYPE)),snapshot)
+  RELEASE_TYPE_IS_SNAPSHOT_OR_EDGE = y
+else ifeq ($(RELEASE_TYPE),edge)
+  RELEASE_TYPE_IS_SNAPSHOT_OR_EDGE = y
+else
+  RELEASE_TYPE_IS_SNAPSHOT_OR_EDGE = n
+endif
+
 ifeq ($(findstring snapshot,$(RELEASE_TYPE)),snapshot)
   # For end-of-week snapshot release, substitute the -SNAPSHOT prefix in the Makefile with
   # the value of $(RELEASE_TYPE) -- assuming value will be snapshot.{X}
   BACKEND_VERSION_TO_RELEASE ?= $(shell semver bump prerel $(RELEASE_TYPE) $(BACKEND_VERSION))
+else ifeq ($(RELEASE_TYPE),edge)
+  # For edge releases, substitute -SNAPSHOT prefix in the Makefile with 'latest'
+  BACKEND_VERSION_TO_RELEASE ?= $(shell semver bump prerel 'latest' $(BACKEND_VERSION))
```

Comment:
I think I didn't explain well myself. This code will push to dockerhub something similar to this:
![image](https://user-images.githubusercontent.com/23639005/52658345-671c0980-2ec0-11e9-9029-603319c22f6b.png)

(this was from my last try). The difference is that this code will put `-latest` suffix instead of `-edge`. But what Travis is currently doing is pushing a `latest` tag without any version attached:

![image](https://user-images.githubusercontent.com/23639005/52658416-929ef400-2ec0-11e9-8aad-11023dd4fae0.png)

I think that what you had previously was correct. It's only the docker tag that should be fixed.

Reply:
Sorry i got it wrong. I just pushed a change.